packaging/services/ovirt-engine/ovirt-engine.xml.in

```diff
@@ -349,11 +357,33 @@
             <valid-connection-checker class-name="org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker"/>
           </validation>
         </datasource>
+        {% if config.getboolean('KEYCLOAK_BUNDLED') %}
+        <datasource
+            jndi-name="java:jboss/datasources/KeycloakDS"
+            pool-name="KeycloakDS"
+            enabled="true"
+            use-java-context="true">
+          <connection-url><![CDATA[{{ config.get('KEYCLOAK_DB_URL') }}]]></connection-url>
+          <driver>postgresql</driver>
+          <pool>
+            <max-pool-size>{{ config.getinteger('KEYCLOAK_DB_MAX_CONNECTIONS') }}</max-pool-size>
+          </pool>
+          <security>
+            <user-name><![CDATA[{{ config.get('KEYCLOAK_DB_USER') }}]]></user-name>
+            <password><![CDATA[{{ config.get('KEYCLOAK_DB_PASSWORD') }}]]></password>
+          </security>
+        </datasource>
+        {% endif %}
 
         <drivers>
           <driver name="postgresql" module="org.postgresql">
             <xa-datasource-class>org.postgresql.xa.PGXADataSource</xa-datasource-class>
           </driver>
+        {% if config.getboolean('KEYCLOAK_BUNDLED') %}
+          <driver name="h2" module="com.h2database.h2">
+            <xa-datasource-class>org.h2.jdbcx.JdbcDataSource</xa-datasource-class>
+          </driver>
+        {% endif %}
```

Comment:
Do we need h2 if Keycloak will use PostgreQL?

Reply:
nope - I'll remove it.

Reply:
done